dpp/src/util/json_value.rs

```diff
@@ -273,6 +278,27 @@ impl JsonValueExt for JsonValue {
             _ => bail!("the Json Value isn't a map: {:?}", self),
         }
     }
+
+    fn replace_base64_paths<'a>(
+        &mut self,
+        paths: impl IntoIterator<Item = &'a str>,
+    ) -> Result<(), Error> {
+        for raw_path in paths {
+            let to_replace = get_value_mut(raw_path, self);
+            match to_replace {
+                Some(v) => {
+                    let base64: String = serde_json::from_value(v.clone())?;
```

Comment:
I just realized that there is a nasty bug in line: https://github.com/dashevo/rust-dpp/blob/0183586f75d421e0c7db30b268a0f392cc870680/dpp/src/util/json_value.rs#L358

It should be Base64 and precisely what you did here. Could you replace the `Bytes` condition with 
```
let base64: String = serde_json::from_value(v.clone())?;
*v = JsonValue::from(base64::decode(base64)?);
```
and use `replace_binary_paths()` instead?, Sorry for the trouble